schema.sql

```diff
@@ -21,6 +21,8 @@ CREATE TABLE owners (
     PRIMARY KEY (id)
 );
 
```

Comment:
- [ ] Please as the last reviewer said create an index for each of the tables, adding indexes for FOREIGN KEYS or on columns frequently used in queries is one of the basic solutions for the performance problems try it out. and ask students to try it out.

`CREATE INDEX ON visits (animal_id);`
this solution may change the time to less than 1000 ms

Reply:
- [ ] @studiosnabeel as previously requested kindy add indexes for the foreign keys or columns used in the queries eg `CREATE INDEX ON visits (animal_id);`, it would be nice if you updated your screenshot when you add the index to showcase the difference.